packages/react-integration/demo-app-ts/src/components/demos/BackToTopDemo/BackToTopDemo.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
It is probably also a good idea to have a demo of the `alwaysShow` case.  Could we also add a test for the interaction?

Reply:
I added a toggle to the demo, let me know if that works!